src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -12,20 +14,25 @@
 public class CaseEventHandlerControllerTest extends SpringBootFunctionalBaseTest {
 
     @Test
-    public void given_ccdEventMessage_should_respond_with_200() {
+    public void given_validEventInformation_should_respond_with_200() {
 
-        CcdEventMessage ccdEventMessage = CcdEventMessage.builder()
-            .id("some id")
-            .name("some name")
+        EventInformation validEventInformation = EventInformation.builder()
+            .eventInstanceId("some event instance Id")
+            .dueTime(LocalDateTime.now())
+            .caseReference("some case reference")
+            .jurisdictionId("IA")
+            .caseTypeId("Asylum")
+            .eventId("submitAppeal")
+            .newStateId("some state id")
+            .userId("some user Id")
             .build();
 
         given()
             .contentType(APPLICATION_JSON_VALUE)
-            .body(asJsonString(ccdEventMessage))
+            .body(asJsonString(validEventInformation))
             .when()
             .post("/messages")
             .then()
-            .statusCode(HttpStatus.OK_200);
-
+            .statusCode(HttpStatus.NO_CONTENT.value());
     }
 }
```

Comment:
Do we need a test to cover the invalidEventInformation scenario? I recall we discussed this but couldn't spot the test in FT on IT?